src/controllers/Event_commentsController.php

```diff
@@ -74,8 +74,8 @@ public function createComment($request, $db)
         $rating = $request->getParameter('rating', false);
         if (false === $rating) {
             throw new Exception('The field "rating" is required', 400);
-        } elseif (false === is_numeric($rating) || $rating > 5) {
-            throw new Exception('The field "rating" must be a number (1-5)', 400);
+        } elseif (false === is_numeric($rating)) {
```

Comment:
Can we remove the ```elseif``` here? There's an exception thrown before so the method is left there already. 

I'd prefer two separate ```if```-statements without the else.